### DISCUSSION TITLE
Question: Is there any timeframe for v1?

### DISCUSSION THREAD

**Discussion by mvanhelmont** [rasmus/EventFlow#857] [Q&A]
Hi All,

Is there any timeframe when v1 gets to beta. We will love to give it a try in our next project. 

Thanks

**Comment by rasmus** [comment]
@mvanhelmont Not yet, the core stuff seems stable, but the packages outside the core are still not converted.

(Sorry, my GitHub issue management abilities are abysmal)

**Comment by rasmus** [comment]
@mvanhelmont I try to push an alpha version of v1.0 during the weekend

**Reply by mvanhelmont** [reply]
thanks will give it a try!

**Reply by rasmus** [reply]
Are there any specific support NuGet packages you are dependent on? I could try to include those... it takes some time to go through each project and convert it to use Microsoft dependency injection and logging

**Reply by mvanhelmont** [reply]
If there is Entity Framework and rabbitmq support for starting it will be great else we will try with inmemory. So don't rush....

**Reply by rasmus** [reply]
@mvanhelmont perfect, I'll see to include those in the next. Version v1.0.4617-alpha was just released
